include/deal.II/base/function_spherical.h

```diff
@@ -0,0 +1,97 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii__function_spherical_h
+#define dealii__function_spherical_h
+
+#include <deal.II/base/config.h>
+
+#include <deal.II/base/function.h>
+#include <deal.II/base/point.h>
+#include <deal.II/base/std_cxx11/array.h>
+
+DEAL_II_NAMESPACE_OPEN
+
+namespace Functions
+{
+
+  /**
+   * The scalar-valued function given in spherical coordinates. The convention
+   * for angles is the same as in GeometricUtilities::Coordinates.
```

Comment:
@bangerth i updated the docs with formula and more explanations. Hopefully it's clearer now.